app/views/users/new.html.erb

```diff
@@ -46,26 +46,18 @@
           %>
         <% end %>
 
-        <div  class="
+        <div class="
                     <%= @person.errors.any? ? 'max-h-40' : 'max-h-0' %> overflow-hidden
                     flex flex-col
                     transition-all duration-100 ease-in
                     space-y-4
                     "
-              data-transition-target="transitionable">
+            data-transition-target="transitionable">
 
-          <%= user_fields.label :first_name, class: "input input-bordered flex items-center justify-between" do %>
-            <span>First name <span class="">*</span></span>
-            <%= user_fields.text_field :first_name,
-              placeholder: "First name",
-              class: "w-44 border-0 dark:text-black"
-            %>
-          <% end %>
-
-          <%= user_fields.label :last_name, class: "input input-bordered flex items-center justify-between" do %>
-            <span>Last name <span class="">*</span></span>
-            <%= user_fields.text_field :last_name,
-              placeholder: "Last name",
+          <%= user_fields.label :name, class: "input input-bordered flex items-center justify-between" do %>
+            <span>Full name <span class="">*</span></span>
+            <%= user_fields.text_field :name,
+              placeholder: "Full name",
               class: "w-44 border-0 dark:text-black"
```

Comment:
This is cool. So the field is “name” and when it attempts to be saved by activerecord it converts “Keith Schacht” and breaks it out into first_name and last_name? That’s nice.

But I think we should change the label. When I read “Full name” I feel like that’s instructing me to type my full name as in first, middle, last. I think maybe a better label is simply “First & Last name”. We may need to adjust the left column slightly if that wraps.

Reply:
> This is cool. So the field is “name” and when it attempts to be saved by activerecord it converts “Keith Schacht” and breaks it out into first_name and last_name? That’s nice.

Exactly. 

Can definitively update the label.

> I assume the gem will magically handle validation even though the text input will be “name” and not “first_name”.

It won't. We need to add validation to `first_name` (model+db) attribute. The gem assumes first name is always present, last name optional, else it breaks.